src/pages/contact/contact.ts

```diff
@@ -1,15 +1,58 @@
 import { Component } from "@angular/core";
+import { AuthHttp } from "angular2-jwt";
+import { Pipe } from "@angular/core";
 
+import { ModalController } from "ionic-angular";
 import { NavController } from "ionic-angular";
 
+import { environment } from "../../environments/environment";
+import { ContactModalPage } from "../modals/contact-modals/contact-modal";
+
 @Component({
   selector: "page-contact",
   templateUrl: "contact.html",
 })
 export class ContactPage {
 
-  constructor(public navCtrl: NavController) {
+  public contacts: any;
+
+  constructor(public navCtrl: NavController,
+              public modalCtrl: ModalController,
+              public auth_http: AuthHttp) {
+    this.auth_http.get(environment.server_url + "/api/user/contact")
```

Comment:
Weird indentation. Should be:
constructor(
&nbsp;&nbsp;public navCtrl: NavController,
&nbsp;&nbsp;public modalCtrl: ModalController,
&nbsp;&nbsp;public auth_http: AuthHttp
)&nbsp;{
&nbsp;&nbsp;&nbsp;this.auth_http....
}